docs/mkdocs/docs/integration/build2/manifest

```diff
@@ -0,0 +1,14 @@
+name: example
+version: 0.1.0-a.0.z
+language: c++
+summary: example C++ executable
+license: other: proprietary ; Not free/open source.
+description-file: README.md
+url: https://example.org/example
+email: your@emailprovider.com
+#build-error-email: your@emailprovider.com
+depends: * build2 >= 0.16.0
+depends: * bpkg >= 0.16.0
+#depends: libhello ^1.0.0
+
+depends: nlohmann-json ^3.0.0
```

Comment:
I think depending from the latest version makes more sense.

Reply:
As long as the last version is 3.x this will acquire the last published version (`^` means any version starting with the major version specified). If we really want to specify any last version then I can remove the version constraint, but it's usually not a good idea.